frontend-react/src/utils/constants.js

```diff
@@ -0,0 +1,4 @@
+export const DIRECTIONS = {
```

Comment:
rtl: true/false is more direct and we can use it e.g. to store value in store, but at the moment when it is passed as a prop to third party library (Material-ui) instead of boolean values we need to use 'ltr' and 'rtl' strings, and to avoid spelling mistakes I prefer to use such enum-like constant.

Reply:
Thanks, resolving.